src/app/shared/form/builder/ds-dynamic-form-ui/models/array-group/dynamic-form-array.component.scss

```diff
@@ -3,7 +3,15 @@
 :host {
   display: block;
 }
-
+.drag-disable {
+  visibility: hidden !important;
+  &:hover, &:focus {
+    cursor: grab;
```

Comment:
Perhaps this should be `cursor: default`? Currently you still see the hand when hovering where the drag handle would be

Reply:
fixed